apm-agent-attach/resources/NOTICE

```diff
@@ -1,3 +1,11 @@
+Elastic APM Java Agent Attach Tool
+Copyright 2018-2019 Elasticsearch B.V.
+
+This product includes software licensed under the Apache License 2.0 from the following sources:
+ - ByteBuddy - Copyright 2014 - 2019 Rafael Winterhalter
+ - https://github.com/kohlschutter/junixsocket - Copyright (c) 2009-2018 Dr. Christian Kohlschütter
```

Comment:
this is a `provided`-scoped dependency so I think we can remove this line as we don't bundle/vendor it